cli/internal/run/run.go

```diff
@@ -745,11 +779,17 @@ func (r *run) executeDryRun(ctx gocontext.Context, engine *core.Scheduler, g *co
 		}
 		sort.Strings(stringDescendents)
 
+		hit, err := turboCache.Assert(hash)
```

Comment:
If we really want to go big here, maybe there's a way to show which cache(s) have the artifact?

Reply:
By that do you mean, fs or http?

Reply:
maybe an object? What about:
```json
"cache": {
  "local": false,
  "remote": true
}
```

or something like that?